.github/workflows/javascript-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Javascript lint
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'web/html/src/**'
+  pull_request:
+    paths:
+      - 'web/html/src/**'
```

Comment:
Do we want to run the linting also when non-javascript files are changed within this folder (it seems there is at least some *.css, *.md, *.json files in there)? Otherwise I would recommend to restrict it here like this: `'web/html/src/**.js'`

Reply:
Thanks for pointing this out, that's a good idea. I've added matchers for all of `.js`, `.jsx`, `.ts` and `.tsx`, I hope I got the syntax correct.